app/screens/backup/Backup.js

```diff
@@ -1,227 +1,44 @@
-  flex: 1;
-  &:first-child {
-    margin-right: 16px;
-  }
-  border-radius: 2px;
-  padding: 24px 18px;
+  justify-content: flex-end;
 `;
 
-const BackupTopWrapper = styled.div`
-  display: flex;
-  flex-direction: row;
-  align-items: flex-end;
-  margin-bottom: 40px;
-`;
-
-const SecurityLogo = styled.img`
-  height: 66px;
-  width: 54px;
-  margin-right: 30px;
-`;
-
-const BackupTitleWrapper = styled.div`
-  display: flex;
-  flex-direction: column;
-`;
-
-const BaseText = styled.span`
-  font-size: 16px;
-  font-weight: normal;
-  line-height: 22px;
-  color: ${smColors.lighterBlack};
-`;
-
-const GrayText = styled(BaseText)`
-  color: ${smColors.darkGray};
-`;
-
-const BackupHeader = styled.span`
-  font-size: 24px;
-  font-weight: bold;
-  line-height: 33px;
-  color: ${smColors.darkGray};
-`;
-
-const RecommendedText = styled.div`
-  font-size: 16px;
-  line-height: 22px;
-  color: ${smColors.darkGray50Alpha};
-  margin-left: 25px;
-`;
-
-const TextContentSection = styled(BaseText)`
-  margin-bottom: 40px;
-`;
-
-// $FlowStyledIssue
-const ActionLink = styled(BaseText)`
-  user-select: none;
-  color: ${({ color }) => color};
-  cursor: pointer;
-  &:hover {
-    opacity: 0.8;
-  }
-  &:active {
-    opacity: 0.6;
-  }
-`;
-
-const content = {
-  file: {
-    text: 'Take your wallet file and save a copy of it in a secure location - We recommend a USB key or a external drive.',
-    note: 'NOTE: You will still need your passphrase to restore the wallet from the file.',
-    icon: shieldIconGreenOne,
-    securityLevel: 'Basic Security',
-    backup: 'File Backup',
-    actionText: 'Show me the wallet file',
-    color: smColors.green
-  },
-  '12words': {
-    text: 'A list of words you keep on a physical paper or in your password manager, that can be used to recover your wallet on any device.',
-    note: 'NOTE: Works even if you lost your passphrase or wallet backup file',
-    icon: shieldIconOrangeTwo,
-    securityLevel: 'Enhanced Security',
-    backup: '12 Words Backup',
-    actionText: 'Create a paper backup',
-    color: smColors.orange
-  }
-};
-
 type Props = {
-  wallet: WalletMeta,
   history: RouterHistory
 };
 
 class Backup extends Component<Props> {
   render() {
+    const { history } = this.props;
     return (
       <Wrapper>
-        <Header>Wallet Backup</Header>
-        <HeaderExplanation>
-          Your wallet file is encrypted with your passphrase on your computer, but We recommend that you’ll backup your wallet for additional security.
-        </HeaderExplanation>
-        <Separator />
-        <SubHeader>How would you like to backup your wallet?</SubHeader>
-        <BackupBoxesWrapper>
-          {this.renderBackupBox('file')}
-          {this.renderBackupBox('12words')}
-        </BackupBoxesWrapper>
-        <ActionLink onClick={this.learnMoreAboutSecurity} color={smColors.green}>
-          Learn more about wallet security and backup
-        </ActionLink>
+        <BackButtonWrapper>
```

Comment:
Fail to see any reason to wrap button instead of giving it position absolute and placing it with left: -15px;

Reply:
Absolute with left -15 will put this component halfway out of the leftmost coordinate of the page. Absolute left 0 is the leftmost coordinate. This can be solved with position absolute but I don't think this is the correct way.

Reply:
You don't have to agree but then reviewer is expecting to see solid arguments to base you disagreement.
My arguments are simpler and easy to control and restyle markup, smaller code base and hight level of competence in final product

Reply:
ok